### PR TITLE
mStable Strategy Adapter

### PR DESCRIPTION
**Pull request by alsco77**
Remaining:

- [ ] 2 of 2 Gnosis safe for proxy upgrades
- [x] Permissions review in VoterProxy/loan repayment (1 of 2 for loan repayment, 2 of 2 for fund movements)
- [x] Strategy tests
- [x] Proxy tests
- [ ] Post-vesting reward distribution calculation

### CODE REVIEW

**Comment by GalloDaSballo** [comment]
I'm getting issues with the compilation of an unrelated file:
(venv) entreprenerd@MacBook-Pro badger-system % brownie test
Brownie v1.13.4 - Python development framework for Ethereum

Compiling contracts...
  Solc version: 0.6.12
  Optimizer: Enabled  Runs: 200
  EVM Version: Istanbul
CompilerError: solc returned the following errors:

deps/@uma/core/contracts/financial-templates/common/financial-product-libraries/FinancialProductLibrary.sol: Warning: SPDX license identifier not provided in source file. Before publishing, consider adding a comment containing "SPDX-License-Identifier: <SPDX-License>" to each source file. Use "SPDX-License-Identifier: UNLICENSED" for non-open-source code. Please see https://spdx.org for more information.

contracts/badger-sett/strategies/native/StrategyCurveLiquidityRewardGaugeBase.sol:247:56: DeclarationError: Undeclared identifier. Did you mean "weth"?
            _swap(lpReward, claimData.lpRewardClaimed, path);
                                                       ^--^

If I delete the file, I still get errors in running tests:

ImportError while loading conftest '/Users/entreprenerd/Desktop/Study/badger-system/tests/conftest.py'.
/usr/local/Cellar/python@3.9/3.9.2_4/Frameworks/Python.framework/Versions/3.9/lib/python3.9/ast.py:50: in parse
    return compile(source, filename, mode, flags,
E     File "/Users/entreprenerd/Desktop/Study/badger-system/tests/conftest.py", line 281
E       if settId == "native.sushiWbtcIbBtc":
E       ^
E   SyntaxError: invalid syntax


I don't believe the PR to contain any tests, but though it would be a good idea to flag up

**Comment by alsco77** [comment]
@GalloDaSballo Yep, seems that there are a few unrelated compilation/testing issues throughout the repo

**Review by sajanrajdev — APPROVED** [review]
LGTM

**Review by sajanrajdev — APPROVED** [review]

**Review by dapp-whisperer — APPROVED** [review]

File: scripts/keeper/harvest.py [hunk 1]
```diff
@@ -59,5 +82,9 @@ def main():
         accounts[0].transfer(badger.keeper, Wei("5 ether"))
         accounts[0].transfer(badger.guardian, Wei("5 ether"))
 
-    skip = keeper_config.get_active_chain_skipped_setts("harvest")
+        skip.append("native.test")
+        skip.append("yearn.wbtc")
+        skip.append("experimental.sushiIBbtcWbtc")
```
[the remark this answers is not in the document]
Comment (sajanrajdev) [reply scripts/keeper/harvest.py:86]: I am not sure to be quite honest but maybe @c0rv0s can clarify this part.

**Pull request merged**